base/operators.jl

```diff
@@ -3,10 +3,27 @@
 ## types ##
 
 """
-    <:(T1, T2)
+```julia
```

Comment:
The signature is not ordinarily a code block

Reply:
I just changed one code block syntax to another code block syntax. The Markdown [docs](https://docs.julialang.org/en/v1/stdlib/Markdown/) mention that the syntax I used should be preferred to the existing syntax:

> "Fenced" code blocks, as shown in the last example, should be preferred over indented code blocks since there is no way to specify what language an indented code block is written in.

Should I disregard that note?

Reply:
The first line of the docstring is a bit special as it's the signature, which is sort of an underspecified pseudo-julia syntax. It's not really supposed to be a code block, it just happens to be rendered that way.